dds/DCPS/AddressCache.h

```diff
@@ -52,7 +53,15 @@ template <typename Key>
 class AddressCache {
 public:
 
+#if defined ACE_HAS_CPP11
+  typedef OPENDDS_UNORDERED_MAP_T(Key, RcHandle<AddressCacheEntry>) MapType;
+  typedef std::vector<Key> KeyVec;
```

Comment:
Use allocator-enabled vector (also below on 62)

Reply:
At one point, I wasn't supporting safety profile here, but obviously not the case anymore.